src/PostgREST/Query/QueryBuilder.hs

```diff
@@ -1,4 +1,5 @@
 {-# LANGUAGE DuplicateRecordFields #-}
+{-# LANGUAGE MultiWayIf #-}
```

Comment:
This is unused right?

Reply:
Yes, was testing something and forgot to remove it.